lingui.config.js

```diff
@@ -1,9 +1,23 @@
 module.exports = {
+  catalogs: [
```

Comment:
I was getting an error about catalogs not being defined after updating lingui. [Lingui Configuration](https://lingui.js.org/ref/conf.html)